modules/mobile/spec/request/maintenance_windows_request_spec.rb

```diff
@@ -107,5 +126,97 @@
         )
       end
     end
+
+    context 'when there are multiple windows for same service with different time spans' do
+      before do
```

Comment:
I think these tests would be slightly clearer for future readers if we instead created these windows to variables (idk if these variable names are accurate):
```
        let!(:earliest_starting) { FactoryBot.create(:mobile_maintenance_evss_first) }
        let!(:middle_starting) { FactoryBot.create(:mobile_maintenance_evss_second) }
        let!(:latest_starting) { FactoryBot.create(:mobile_maintenance_evss_third) }
```

Then your tests can test again those values like:
`expect(attributes.pluck('startTime').uniq).to eq([earliest_starting.start_time])`

It makes the tests more self-documenting.

Reply:
sounds good. updated